lib/twilito/api.rb

```diff
@@ -31,14 +31,13 @@ def messages_uri(account_sid)
 
     private
 
-    def twilio_params(args)
-      {
-        'To' => args[:to],
-        'From' => args[:from],
-        'Body' => args[:body],
-        'MediaUrl' => args[:media_url],
-        'StatusCallback' => args[:status_callback]
-      }.compact
+    # NOTE: Converts snake_cased hash of arguments to CamelCase to match Twilio
+    # API expectations. Also, removes auth_token and account_sid as those are
+    # included separately in .send_response as basic auth instead of POST body
+    def twilio_form_data(args)
+      args
+        .merge(auth_token: nil, account_sid: nil).compact
+        .reduce({}) { |result, (k, v)| result.merge(k.to_s.split('_').collect(&:capitalize).join => v) }
```

Comment:
This would be easier to read with ActiveSupport (`Hash#except` and `Hash#transform_keys`, especially) but I really do want to put off adding any runtime dependencies to Twilito as long as possible.